### PR TITLE
Made questions dynamic via tables and appropriate fetches

### LINKED ISSUE #103: Making Questions Dynamic
This code can be cleaned up. Ideally we want to have an arbitrary number of questions. Right now the code is indexing into an array that is strictly size 4.

https://github.com/codebase-berkeley/teacher-portal/blob/c21efc077f3daf8fca82a5769c536dd435e2803e/server/routes/index.js#L88-L135

### PR DESCRIPTION
**Pull request by rrrliu**
Fixes #103

### CODE REVIEW

File: server/routes/index.js [hunk 1]
```diff
@@ -66,7 +66,10 @@ router.get('/studentSummary/:unitID', async (req, res) => {
     );
     const data = [];
     const { rows } = mainquery;
-
+    const qquery = await db.query(
```
Comment (partyshah) [inline server/routes/index.js]: change this to a better variable name

File: server/routes/index.js [hunk 2]
```diff
@@ -80,35 +83,47 @@ router.get('/studentSummary/:unitID', async (req, res) => {
         }
       });
 
-      const q = [[0], [0], [0], [0]];
+      const q = [];
+
+      for (let i = 0; i < NUMQS; i += 1) {
```
Comment (partyshah) [inline server/routes/index.js]: don't do two for loops please. this can be made into one.

Comment (rrrliu) [reply server/routes/index.js]: can't be resolved easily

File: server/routes/index.js (hunk 1 above)
Comment (rrrliu) [reply server/routes/index.js]: changed code

**Pull request merged**